### DISCUSSION TITLE
Frontend Initiatives Meeting - April 27, 2021

### DISCUSSION THREAD

**Discussion by varl** [dhis2/notes#272] [Meetings] [answered]
When: April 27, 2021 @ 15:00 CET
Where: Remote

# Agenda

-   General updates
    -   Add the [Frontend Group calendar](https://calendar.google.com/calendar/embed?src=c_aebpolkmfvanjcfgf8f96bnbi8%40group.calendar.google.com&ctz=Europe%2FOslo): `c_aebpolkmfvanjcfgf8f96bnbi8@group.calendar.google.com`
    -   Meetings and other frontend meetings will be added to that calendar (e.g. UI, App Platform, etc.).
        - Feel free to join those meetings if you are interested, or check the notes if you just want to see what was discussed and decided.
    -   Other calendars you should subscribe to:
        -    Release calendar: `dhis2.org_vhltgg78od75pqm4m39t7arvc8@group.calendar.google.com`
        -    Core dev calendar: `dhis2.org_tbep59bdcpc5k0rl2c8ctp962k@group.calendar.google.com`
        -    Dev holiday calendar: `dhis2.org_a5jeon529ng4ad8srv3ajlnp0s@group.calendar.google.com`
-    Recruitment
        -    Who wants to help with recruitment?

**Comment by Mohammer5** [comment]
Should we talk about this one? https://github.com/dhis2/app-platform/pull/538
Or is it too early and we should talk about it once it's implemented/available?

**Comment by edoardo** [comment]
Status on https://github.com/dhis2/ui/pull/463 since I'm working on rewriting the `FavoritesDialog` and the new design calls for the data table component.

**Comment by varl** [comment, marked as answer]
Meeting minutes draft: https://github.com/dhis2/notes/pull/281
